Update KCS Comms Team Handbook (#7961)
kubernetes/community: events/events-team/comms/README.md

# Communications Team Handbook

This document defines communications activities needed to run a

> Kubernetes Contributor Summit (KCS). The Comms Team consists of 1-2 lead(s), shadows, volunteers, at least 1 emeritus lead, and a Website Manager.

Reviewer: This is for the comms team for contributor summits. Yes, it's slightly confusing.

> Comms Leads are responsible for leading the comms team, training shadows, and ensuring all comms activities are planned, prioritized, and completed successfully. They are committed to run the comms team for the current cycle and advise as Emeritus KCS Comms Lead in the following comms cycle (2 cycles, 1 year). Must have [Kubernetes Org Membership]. Expected to attend in-person.

Reviewer: > 2 cycles, 1 year

Could we say 2 cycles per year? Is that accurate with India being in the mix now too or is someone else running with it?

Reviewer: Also, cycle, I wasn't clear if that meant EU and US but I'm assuming so.

> Shadows will learn both how to do comms and how to run the KCS Comms team, as directed by the Comms Lead. They are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). Must either have [Kubernetes Org Membership] or be planning to pursue org membership.

Reviewer: I would change "persue" to be, "on their way towards org membership"

I don't want to exclude anyone, but at the same time a shadow that's not an org member; feels like that'd be harder than expected. Have we ever granted org membership to someone that is helping with the current comms cycle? cc @jberkus

Reviewer 2: We've given people a bump, but not if KCS Comms was their very first contribution.  So +1 on "in the process of applying for org membership".

Author: A couple of the comms shadows this cycle got their org membership during the cycle.

> Volunteers are expected to assist with comms during the current KCS cycle, as directed by the lead. Volunteers do not need to have or be pursuing [Kubernetes Org Membership].

Reviewer: Volunteers are GREAT!

But, for KCS Comms is it okay to bring in someone completely new to the project to help? What would that work look like? I think answering these two questions will need to be in the doc (an FAQ or something like that is an option too).

Reviewer 2: Sure, why not?  It's a pretty good first contribution task.

Author: +1. One problem with comms is just the sheer volume. If someone wanted to pop in to help with getting some drafts written, but didn't want to commit to being on a leadership path, I would still welcome the help.

> An emeritus comms lead is a past lead who provides mentorship to the current comms lead. Should have [Kubernetes Org Membership].

Reviewer: Should to Must

If someone is an emeritus comms lead, I would hope that we'd grant org membership as soon as we could. But, what examples were you thinking of here?

Reviewer: So the requirements below says the "KCS Comms Lead must have \[Kubernetes Org Membership\]. (They should not plan to pursue membership during the cycle.)"  
  
Then we definitely update this language for org membership as a requirement.

Reviewer 3: I guess we could have the case, someone lost the membership because of inactivity and came back and picked up by supporting. But i don't know if this ever happened

Author: I was thinking of a case like Mario is describing here. Like I don't know if Paris' org membership is still active, but imagine it wasn't and we needed help and she volunteered to serve as an emeritus advisor. I'd still accept that. Unlikely. But possible.

The Website Manager is responsible for ensuring updates to the [Kubernetes Contributor Website](k8s.dev) happen. Requires [Kubernetes Org Membership]

- [KCS Comms Lead Overview](#kcs-comms-lead-overview)
- [Shadow to the KCS Comms Lead Role](#shadow-to-the-kcs-comms-lead-role)
- [KCS Comms Volunteer Overview](#kcs-comms-volunteer-overview)

> - [KCS Website Manager Overview](#kcs-website-manager-overview)

Reviewer 4: This anchor `KCS Website Manager Overview` isn't working.

> - [Emeritus KCS Comms Lead Overview](#kcs-website-manager-overview)

Reviewer 4: This anchor `Emeritus KCS Comms Lead Overview` isn't working.

- [Comms Rules and Guidelines](#comms-rules-and-guidelines)
- [Documentation](#documentation)

## KCS Comms Lead Overview

> A KCS Comms Lead should plan to be involved with KCS planning for at least a year (2 KCS events), first as lead (for 1 or more events), and then as an emeritus mentor to the succeeding Comms lead (for 1 or more events). The workload of being an Emeritus Comms Lead is light, but they are still actively involved.

Reviewer: > 2 KCS events

Events or cycles; let's keep the language consistent.

> A KCS Comms Lead holds that role from the beginning of planning for a specific KCS event until all activities for that event are completed, including post-event activities such as event retros and updating the handbook (this doc). This timeframe is the "cycle" for that KCS event.

Reviewer: If we stick with the cycle term then this paragraph really needs to be before the first mention of a cycle.

As Communications Lead, you are responsible for the overall communications
before, during, and after the event. The role also includes ensuring all non-technical content (web, mail,
physical prints) is correct and informative, and supporting the other roles
(registration, content) as needed.

> A Comms Lead should plan to attend their KCS event in-person. Onsite, a Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur onsite. In the event that a KCS Comms Lead is unable to attend in-person, the lead is responsible for designating an in-person comms point of contact.

Reviewer 5: ```suggestion
A Comms Lead should plan to attend their KCS event in-person. A Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur during the event. In the event that a KCS Comms Lead is unable to attend in-person, the lead is responsible for designating an in-person comms point of contact.
```
Onsite was repeated thrice in the same sentence.

> A Comms Lead should plan to attend their KCS event in-person. Onsite, a Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur onsite. In the event that a KCS Comms Lead is unable to attend in-person, the lead is responsible for designating an in-person comms point of contact.

Reviewer 3: ```suggestion
A Comms Lead should plan to attend their KCS event in-person. Onsite, a Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur onsite. In the event that a KCS Comms Lead is unable to attend in-person, the lead is responsible for designating an in-person comms point of contact, that have access to the various comms channels. 
```

The person on site need to act fast so they should have access or rights to publish

When their cycle is complete, a KCS Comms Lead becomes "emeritus." The most recent Emeritus Comms Lead should plan to be actively involved in the following KCS cycle as a mentor to the next KCS Comms Lead.

Time Commitment:
- 1-3 hours a week from 0-1.5 months in
- 3-5 hours a week from 1.5 months-to event
- 1-3 hours post-event for retros and handbook updates
- 1-2 hours per month through the following KCS cycle.

### Skills and Qualifications

> - A KCS Comms Lead must have [Kubernetes Org Membership](). (They should not plan to pursue membership during the cycle.)

Reviewer 6: ```suggestion
- A KCS Comms Lead must have [Kubernetes Org Membership]. (They should not plan to pursue membership during the cycle.)
```

- A KCS Comms Lead should plan to attend their KCS event in-person
- Successful completion of a past cycle as a KCS Comms Shadow, or equivalent experience
- Good grasp on general comms activities surrounding events
- Understanding of common forms of contributor communications including Slack, email, and contributor-owned social media accounts
- A healthy emoji vocabulary!

### Responsibilities

- Create a Comms Team plan for the cycle
  - Create a KCS Comms Team plan using the [KCS Comms Team Plan Template]
    - Shadow requirements
    - Volunteer requirements
    - Associated activities such as comms and meeting updates
    - Update the included rough comms timelines

> - Create a doc for comms drafts using the [KCS Comms Draft Doc Template]

Reviewer 4: Do we want to mention about Award / logo designs?

Author: This should be owned by the Design lead. Which does not currently exist, but should in the future. I'll need to create a new role handbook.

  - This includes email, Slack, and social media (via the Buffer platform with Contributor Comms)
  - You'll source the informaton needed from your fellow leads, and can refer to prior art (linked below)
- Manage online presence of the website, alongside the social media strategy
- Recruit a website lead to handle all updates to the website
  - Consider ensuring website reviewers/approvers are on the team to merge upstream changes
- Provide updates on social media, mailing lists, and Slack before, during, and after the event
  - After-event communications will include a post-event survey, created by SIG ContribEx leads
- Work alongside the CNCF Events Team to coordinate communications efforts
- Assist with deck content for curated talks and/or other purposes

> - Onsite, a Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur onsite.

Reviewer 5: ```suggestion
- A Comms lead acts as the contact person for any onsite communications, such as messaging about room or registration changes that may occur during the event.
```

### What to do as KCS Comms Lead
This section acts as a checklist for activities the KCS Comms Lead is likely to do each cycle.

- Create Slack DM with Comms team (lead, shadows, volunteers)

> - Create a doc using the [KCS Comms Team Plan Template] & put it in the relevant KCS folder

Reviewer: Can we link to the Comms folder here?

  - Update shadow & volunteer requirements to suit your team
  - Update Staff Summit Meeting sign up sheet
- Create Email Plan Doc & put it in the relevant KCS folder
- [Optional] Set up meetings with Comms team
- Claim and write the announcement comms (email &/or social, slack, etc.)
- Share Plan Docs with team
  - Ensure shadows and volunteers are signed up for necessary activities
- Ensure comms leading up to the event are planned, prioritized, and completed successfully
- [Post-KCS] Update role handbook
- [Post-KCS] Update KCS Comms Overall Plan Template doc

## Shadow to the KCS Comms Lead Role

> Shadows are expected to serve as KCS Comms Leads in a future cycle. During their shadowing cycle(s), Shadows will work closely with the KCS Comms Lead to learn both how to run the KCS Comms team and how to create comms themselves. By signing up to shadow, you are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). A Shadow candidate must either have [Kubernetes Org Membership] or be planning to pursue org membership.

Reviewer 4: ```suggestion
Shadows are expected to serve as KCS Comms Leads in a future cycle. During their shadowing cycle(s), Shadows will work closely with the KCS Comms Lead to learn both how to run the KCS Comms team and how to create comms themselves. By signing up to be a shadow, you are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). A Shadow candidate must either have [Kubernetes Org Membership] or be planning to pursue org membership.
```

[Kubernetes Org Membership] can be obtained through KCS Comms work, and a shadow should complete enough tasks to be eligible for org membership over the course of a cycle. Do note that org membership is an indication of active and continuous contribution to the Kubernetes project/community. Shadows are expected to be committed to contributing to Kubernetes on a long-term basis.

### Requirements
* Review this document.

> * A KCS Comms Shadow must either have, or plan to pursue, [Kubernetes Org Membership].

Reviewer: I'd like to see at least one commit before they shadow. Non-org member shadows could be asked to step into the lead role in case the Lead has to drop out for reasons outside their control.

I have no idea how to phrase that though; happy to workshop the language.

Author: I don't think Arvind or Sandeep had any commits before shadowing this time.

* Commit to contributing to the KCS Staff team for multiple cycles.

> ## Tasks

Reviewer 5: ```suggestion
### Tasks
```

During the course of their KCS cycle as a shadow, KCS Comms Shadows are expected to complete tasks including but not limited to:
* Joining the appropriate Slack channels for KCS staff collaboration and for sending comms about KCS to the community
* Working with the Comms Lead and Contributor Comms Subproject to get access to tooling for posting on Kubernetes Contributor-focused social media channels.
* Representing the KCS Comms team in KCS Staff meetings.
* Writing email comms as instructed by the KCS Comms Lead.
* Writing social media comms as instructed by the KCS Comms Lead.

> * Reviewing comms written by other shadows, volunteers, or the lead as directed by the KCS Comms Lead

Reviewer: I would drop the, "as directed..." part here as I think it's a must for even the leads' work to be reviewed.

Shadows should work with the Comms Lead to ensure they are getting sufficient experience to feel comfortable taking on the KCS Comms Lead role in a future cycle after completion of a shadow cycle.

> In the event a shadow is unable to complete the tasks expected of them as designated by the KCS Comms Lead, they should inform the KCS Comms Lead as early as possible, to the best of their ability. Shadows who are unable to complete the required tasks as designated by the KCS Comms Lead, will be removed from the list of shadows. Upon failure to complete a shadowing cycle, the contributor may request to become a shadow in a future cycle. Successful completion of a shadowing cycle (as determined by the KCS Comms Lead) or equivalent experience is required to become a KCS Comms Lead.

Reviewer: I would actually bold the, "as early as possible." And maybe throw in some language that life happens and we support you in your endeavours be they here or elsewhere.

## KCS Comms Volunteer Overview
Volunteers are expected to assist with comms during the current KCS cycle, as directed by the lead. Volunteers do not need to have or be pursuing [Kubernetes Org Membership].

### Requirements
* Review this document.
* Commit to supporting KCS Comms through the current cycle

> ### Tasks

Reviewer: Is the Lead allowed to delegate to a shadow to management of volunteers? I feel like we might be putting too much burden on the lead potentially.

Author: A volunteer could just pop in to help with a single message. They're not training for leadership, so they don't need a path or work assigned to them. They can contribute or not. We haven't had anyone volunteer yet at this point, so hard to say exactly what a volunteer might want out of the experience or how much management they might need.

During the course of their KCS cycle as a volunteer, KCS Comms Volunteers are expected to complete tasks including but not limited to:
* Joining the appropriate Slack channels for KCS staff collaboration and for sending comms about KCS to the community
* Working with the Comms Lead and Contributor Comms Subproject to get access to tooling for posting on Kubernetes Contributor-focused social media channels.
* Writing email comms as instructed by the KCS Comms Lead.
* Writing social media comms as instructed by the KCS Comms Lead.

> * [optional] Reviewing comms written by other volunteers, shadows, or the lead as directed by the KCS Comms Lead

Reviewer: I would change optimal to encouraged

> ## KCS Website Manager

Reviewer: This new role is a GREAT idea.

I think it's possible that non-KCS planning members could help here too (case in point, folks with Hugo and/or Docsy experience).

Might even be worth it to say they can lean on the Contributor Comms team for assistance too, as needed.

The KCS Website Manager is expected to collaborate with all KCS Staff leads and their teams to understand what changes need to be made to the website, when those changes need to go live, and ensuring that all KCS-related website changes are planned, prioritzed, and completed.

### Requirements
- Review this document.
- [Kubernetes Org Membership]
- Commit to supporting KCS Comms through the current cycle and the following cycle as emeritus Website Manager (1 year, 2 cycles)
- Basic familiarity with the [Kubernetes Contributor Site (k8s.dev)](https://github.com/kubernetes/contributor-site), especially the events section (k8s.dev/events)
- Basic familiarity with the [Hugo website framework](https://gohugo.io/)
- Basic familiarity with the [Markdown](https://www.markdownguide.org/) markup language

### Tasks

> During the course of their KCS cycle as a website manager, KCS Website Managers are expected to complete tasks including but not limited to:

Reviewer: Can there be multiple website managers (the term webmaster from the 90s did pop into my head, but that's only because I was one in high school).

Reviewer 2: Theoretically, yes, but in practice we've struggled to recruit even one.

- Joining the appropriate Slack channel(s) for KCS staff collaboration
- Creating the page for the specific event within k8s.dev/events
- Adding and maintaining information and pages throughout the event cycle, including:
  - Registration info
  - CFP & schedule info

>   - Social venue and registration info

Reviewer 3: ```suggestion
  - Social venue and registration info
  - Notes folder
  - Unconference informations
```

  - and more

## Emeritus KCS Comms Lead Overview
After completing a cycle as a KCS Comms Lead, the former lead takes on emeritus status. As an Emeritus KCS Lead, the former lead is expected to mentor the following KCS Lead(s). The Emeritus Lead can provide mentorship for as long as they like, but it's especially valuable for the previous lead to mentor their direct successor.

### Requirements
- Has completed at least 1 cycle as a KCS Comms Lead
- [Kubernetes Org Membership]

### Tasks
- Provide mentorship to the KCS Comms Lead

## Comms Rules and Guidelines

### Social Media Rules

> - Send KCS comms to k8scontributors accounts on whatever platforms they exist on. At time of writing, that’s “X” and Mastodon. Social media should be sent only to contributor-facing accounts.

Reviewer: > At time of writing, that’s “X” and Mastodon. Social media should be sent only to contributor-facing accounts.

I would just link to the contributor comms social media directory and stay away from specifics on which network is in use.

>   - Kubernetesio on X and the Kubernetes Linkedin accounts should not be used because their audiences include end users and the general public..

Reviewer 7: - Kubernetesio on X and the Kubernetes LinkedIn accounts should not be used because their audiences include end users and the general public.

>   - Kubernetesio on X and the Kubernetes Linkedin accounts should not be used because their audiences include end users and the general public..

Reviewer 7: - Kubernetesio on X and the Kubernetes LinkedIn accounts should not be used because their audiences include end users and the general public.

> - Make sure comms are formatted in Arial, 11pt, with single line spacing before copying into the target platform. This will ensure that the text looks consistent wherever it is posted.

Reviewer: I would again stay away from specifics here and replace it with a more general guideline of keeping the doc consistent. As screens increase pixel density, 11 pt might not be big enough.

> - If you are scheduling comms to go out at a specific time, make sure that time is aligned with the timezone the event will be held in.

Reviewer: This should be a top level thing not buried on line 166.

> - Link to k8s.dev/events and subpages, or email messages, wherever possible. Avoid sharing links to contributor-specific links such as Registration, CFP, and GitHub Issues.

Reviewer: Again, top level guidance for the whole team.

  - While our Social Media presence is largely targeted at contributors, all social media posts are public. As such, we should avoid sharing links which could cause unwanted spam/interactions.

> - Review the Contributor Comms team’s Social Media Guidelines.

Reviewer: A link here would be helpful.

- All social media should have at least 1 review, preferably by a shadow. A review by a social media approver is a given.
- Generally, social media should be ready for review at least 1 day before it needs to be sent.

> - Generally, an email should be ready for review around 3 or more days before it needs to be sent.

Reviewer 5: ```suggestion

```
Should we leave this line out about email reviews from this section as it should be (and is) covered in the section of Email Tips and Rules?

- If any shadow does not have Buffer access, let the KCS Comms lead know.
- Always consider your intended and actual audience! When a message is only applicable to org members (which is pretty much - everything for the contributor summit), state that clearly!

> 

Reviewer 5: ```suggestion
```
(nit - an extra line) Not sure if it'd impact the final markdown version so pointing it out just in case :)

> ### Email Tips and rules

Reviewer 7: ### Email Tips and Rules

- NOTE: A sender’s first email to a mailing list will get stuck in the moderation queue. Include around a day in the delivery estimate to account for the time a sender’s first email will spend in the moderation queue. Shadows & Volunteers should plan to send their first email to a given mailing list (leads, dev, etc) early in the day on the day it’s due. (per the timezone of the location where the summit will be held).

> - Make sure comms are formatted in Arial, 11pt, with single line spacing before copying into the target platform. This will ensure that the text looks consistent wherever it is posted.

Reviewer: Since it's mentioned twice, what's the problem we're trying to prevent here?

Reviewer 5: Regarding the moderation queue, we added this information to the comms document after my first email got stuck in the queue. At the time, we didn't realize this and thought the email hadn't been sent. It took some time to reach out to the leads and ask them to approve the email, so we mentioned this to account for the delay during a shadow's first email.

On the formatting specifics, we included these guidelines because, when my first email was sent out, it looked fine in the document. However, after sending, we noticed inconsistencies in styling. This happened because I drafted the email in my personal doc and copied it over to the comms doc, which was initially fine, but adding extra information directly into the draft in the comms doc led to formatting issues. These were not very noticeable in the Google Doc, and I missed them. Therefore, we thought it would be helpful to specify formatting details (Arial, 11pt, single line spacing) to ensure consistency in all communications.

However, I agree that we could replace these specifics with a more general guideline and perhaps use this example to illustrate the point.

- An email should be ready for review around 3 or more days before it needs to be sent.

> - Send a message to the KCS Comms group dm (this is usually a dm, not a channel) on the #summit-staff channel in Slack when an email is ready for review, and preferably 1 day before it goes out.

Reviewer 7: - Send a message to the KCS Comms group DM (this is usually a DM, not a channel) on the #summit-staff channel in Slack when an email is ready for review, and preferably 1 day before it goes out.

> - Send a message to the KCS Comms group dm (this is usually a dm, not a channel) on the #summit-staff channel in Slack when an email is ready for review, and preferably 1 day before it goes out.

Reviewer 5: ```suggestion
- Send a message to the KCS Comms group dm (this is usually a dm, not a channel) and on the #summit-staff channel in Slack when an email is ready for review, and preferably 1 day before it goes out.
```

Reviewer: Announce bot?
